### DISCUSSION TITLE
Fill all struct fields

### DISCUSSION THREAD

**Discussion by njkevlani** [hrsh7th/nvim-cmp#983] [Q&A]
Is it possible to fill all struct fields with completion?

Like `Fill all fields...` shown [here](https://www.youtube.com/watch?v=N0jvAea46YM&t=838s).

**Comment by smjonas** [comment]
I think this depends on whether your LSP server supports it. Probably a snippet will be provided when you type the function name in this case. Make sure to use https://github.com/hrsh7th/cmp-nvim-lsp as well.
